src/Settings/Path.hs

```diff
@@ -146,6 +150,12 @@ packageDbDirectory :: Stage -> FilePath
 packageDbDirectory Stage0 = buildRootPath -/- "stage0/bootstrapping.conf"
 packageDbDirectory _      = "inplace/lib/package.conf.d"
 
+-- | ( During installtion) Path to package database directory of a given 'Stage'.
```

Comment:
Ah, I missed the fact that now there are two different `packageDbDirectory`-like paths! Anyway, I think this will benefit from more comments.